agenthub/langchains_agent/utils/llm.py

```diff
@@ -88,60 +93,73 @@
 You can also use the same action and args from the source monologue.
 """
 
-class Action(BaseModel):
+
+class _ActionDict(BaseModel):
     action: str
     args: dict
 
```

Comment:
Currently, this `_ActionDict` is just a hack for compatibility with the old code to make sure langchains agent runs smoothly. Maybe eventually we will replace this with the actual `Action` in this low-level implementation.